### ISSUE TITLE
Money Transfer Report

### ISSUE THREAD

**Issue by cadeath** [cadeath/Dalton#21]
Payout = Send Out
PayIn = Send In

replace Send Out by Send In

Need to Enumerate reports with Term Problems

Credit to: FROI - ROXAS

**Comment by cadeath** [comment]
MONEY_TRANSFER VIEW
    CASE MoneyTrans 
      WHEN 0 THEN 'PAYOUT'
      WHEN 1 THEN 'SENDOUT'
      ELSE 'NA'
    END AS MoneyTrans,
Line # 16 Should be SEND IN

**Comment by cadeath** [comment]
CASE MoneyTrans
      WHEN 0 THEN NETAMOUNT 
      ELSE 0 END AS PAYOUT,
    CASE MoneyTrans
      WHEN 1 THEN NETAMOUNT 
      ELSE 0 END AS SENDOUT,
    ServiceCharge,
Line # 15 & 18 should be AMOUNT not NETAMOUNT

**Comment by cadeath** [comment]
For DB Update
```
SELECT 
	ServiceType, TransDate,
    CASE MoneyTrans 
      WHEN 0 THEN 'PAYOUT'
      WHEN 1 THEN 'SENDIN'
      ELSE 'NA'
    END AS MoneyTrans,
    CASE
      WHEN ServiceType = 'Pera Padala' AND MoneyTrans = 0
      THEN 'ME# ' || LPAD(TransID,5,0)
	  WHEN ServiceType = 'Pera Padala' AND MoneyTrans = 1
      THEN 'MR# ' || LPAD(TransID,5,0)
      ELSE RefNum END as RefNum,
    CASE MoneyTrans
      WHEN 0 THEN AMOUNT 
      ELSE 0 END AS PAYOUT,
    CASE MoneyTrans
      WHEN 1 THEN AMOUNT 
      ELSE 0 END AS SENDOUT,
    ServiceCharge,
    Commission,
    NETAMOUNT
FROM
	TBLMONEYTRANSFER
WHERE
	Status = 'A';
```

**Comment by cadeath** [comment]
FOR REVIEW

**Comment by cadeath** [comment]
DB Update
1.0.12

> CREATE VIEW MONEY_TRANSFER(
>   SERVICETYPE,  TRANSDATE,  MONEYTRANS,
>   REFNUM,  PAYOUT,  SENDOUT,  SERVICECHARGE,
>   COMMISSION,  NETAMOUNT,  SENDERNAME,
>   S_ADDR,  RECEIVERNAME,  R_ADDR,  LOCATION)
> AS
> SELECT 
>   MT.ServiceType, MT.TransDate,
>     CASE MoneyTrans 
>       WHEN 1 THEN 'PAYOUT'
>       WHEN 0 THEN 'SENDIN'
>       ELSE 'NA'
>     END AS MoneyTrans,
>     CASE
>       WHEN ServiceType = 'Pera Padala' AND MoneyTrans = 0
>       THEN 'ME# ' || LPAD(TransID,5,0)
>     WHEN ServiceType = 'Pera Padala' AND MoneyTrans = 1
>       THEN 'MR# ' || LPAD(TransID,5,0)
>       ELSE RefNum END as RefNum,
>     CASE MoneyTrans
>       WHEN 1 THEN AMOUNT 
>       ELSE 0 END AS PAYOUT,
>     CASE MoneyTrans
>       WHEN 0 THEN AMOUNT 
>       ELSE 0 END AS SENDOUT,
>     MT.ServiceCharge,
>     MT.Commission,
>     MT.NETAMOUNT, MT.SENDERNAME, 
>     C.ADDR_STREET || ' ' || C.ADDR_BRGY || ' ' || C.ADDR_CITY AS S_ADDR, 
>     MT.RECEIVERNAME, 
>     R.ADDR_STREET || ' ' || R.ADDR_BRGY || ' ' || R.ADDR_CITY AS R_ADDR,
>     MT.LOCATION
> FROM 
>   TBLMONEYTRANSFER MT
>   INNER JOIN TBLCLIENT C
>   ON  MT.SENDERID = C.CLIENTID
>   INNER JOIN TBLCLIENT R
>   ON  MT.RECEIVERID = R.CLIENTID
> WHERE 
>   Status = 'A';

BUG FOR THE REPORT, INTERCHANGE PAYOUT AND SEND IN RESULT

**Comment by cadeath** [comment]
Included in v1.1.2.5

**Comment by cloudNine09** [comment]
DIGOS2 : 
WESTERN UNION OUT TRANSACTION APPEARS IN THE TRANSACTION TYPE OF REPORT AS SEND IN AND PERA PADALA IN TRANSACTION APPEARS AS PAYOUT.

**Comment by cadeath** [comment]
Payout and Send In is fixed in version 1.2.2
The TAG was interchange
DB tag 0 means Send In
DB tag 1 means Pay Out

For review
